src/store/creator/reducer.ts

```diff
@@ -1,9 +1,10 @@
 import { createReducer } from '@reduxjs/toolkit';
 import { CreatorState, CreatorStatus } from './types';
 import { createAnnotationAction } from '../annotations';
-import { setStagedAction, setStatusAction, setCursorAction } from './actions';
+import { fetchCollaboratorsAction, setStagedAction, setStatusAction, setCursorAction } from './actions';
 
 export const initialState = {
+    collaborators: null,
```

Comment:
What are your thoughts on moving collaborators state out of creator and into it's own slice?

Reply:
I agree. It looks like this data is already tracked in a `users` reducer, separately.